src/exercise-sheet-comment/exercise-sheet-comment.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/PrismaService';
+import { CreateExerciseSheetCommentInput } from '../graphql/graphqlTypes';
+import { User } from '@prisma/client';
+
+@Injectable()
+export class ExerciseSheetCommentService {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async create(input: CreateExerciseSheetCommentInput, user: User) {
+    return this.prismaService.exerciseSheetComment.create({
+      data: {
+        comment: input.comment,
+        userId: user.id,
```

Comment:
For connecting different tables, this is fine, but I prefer the following syntax (Check: https://www.prisma.io/docs/orm/prisma-client/queries/relation-queries)


```
{
    data: {
        user: {
            connect: {
               ... // Here the IDE autocorrects, I don't remember rn xD
           }
      }
```

Also add the ability to specify contributors. This gql input type currently doesn't support contributors